src/public/modules/forms/admin/components/react-switch-banner.client.component.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const AdminService = require('../../../../services/AdminService')
+const { UiCookieValues } = require('../../../../../../shared/types')
+angular.module('core').component('reactSwitchBannerComponent', {
+  templateUrl:
+    'modules/forms/admin/componentViews/react-switch-banner.client.view.html',
+  controller: ['$scope', '$q', '$window', reactSwitchBannerController],
+  controllerAs: 'vm',
+})
+
+function reactSwitchBannerController($scope, $q, $window) {
+  const vm = this
+  $scope.ui = UiCookieValues.React
```

Comment:
i don't think this is needed

Reply:
by this do you mean I can replace `$scope` with `vm`?